pyoptsparse/postprocessing/OptView.py

```diff
@@ -747,7 +747,7 @@ def save_tec(self):
             for name in values:
                 dat[name] = self.var_data[name]
 
-        keys = dat.keys()
+        keys = list(dat.keys())

```

Comment:
Given that this is unsorted, are we sure that for example line 753 is correct?

Reply:
This should be fine. Although dat.keys() are not sorted, all its arrays have the same size. I have tested it with more than one key and it worked fine.

Reply:
Ah okay gotcha.

Reply:
BTW: I think the save tec file is a very useful function for OptView. However, it is not in the OptView_dash version. I can not save the optimality to files for plotting. Maybe save_tec_file is a new feature worth migrating to OptView_dash

Reply:
Yeah OptView in general is a bit of a mess. We should clean that up, modernize it and make sure everything's consistent, but we just haven't had the chance to.